### PR TITLE
add dropdown demo in showcase

### PR DESCRIPTION
**Pull request by houfeng0923**

### CODE REVIEW

File: examples/showcase/dropdown/we-dropdown.we [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    }
+
+    .current {
+        color: #0088FB;
+    }
+</style>
+
+
+<script>
+    require('weex-components');
```
Comment (luics) [inline examples/showcase/dropdown/we-dropdown.we]: pls remove `weex-components`  if  not necessary.

File: examples/showcase/dropdown/dropdown-demo.we [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    .status-score { color: #666; }
+    .status-confirmed { color: #090; }
+    .status-feedback { color: #666; }
+    .status-wait_confirm { color: #f90; }
+    .status-wait_feedback { color: #f00; }
+
+</style>
+
+<script>
+    require('weex-components');
```
Comment (luics) [inline examples/showcase/dropdown/dropdown-demo.we]: pls remove `weex-components`  if  not necessary.

**Comment by houfeng0923** [comment]
@Jinjiang   .   this demo is aim to show the dropdown on first push;  now I have added the filtering feature.

**Comment by Jinjiang** [comment]
Awesome! +1 @luics

**Comment by luics** [comment]
nice job!

**Pull request merged**